src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Opening Keynote with Dr. Arun Majumdar</Subtitle>
+                    <EventWithDateInfo date="APR 3 1 PM EST"
+                                        description={['Founding Director @', 'ARPA-E https://arpa-e.energy.gov/',
+                                                    '\n Former VP, Energy @', 'Google https://google.com/',
+                                                    '\n Co-Director, Precourt Institute for Energy @', 'Stanford https://energy.stanford.edu/']}/>
+                </EventWithDate>
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Find a Team</Subtitle>
+                    <EventWithDateInfo date="APR 3 2 PM EST"
+                                        description={['Speed-dating, but for climate co-founders.']}/>
+                </EventWithDate>
+            </EventGroup>
+
+            <EventGroup>
+                <Subtitle fontColor={GREEN}>Workshop Your Proposal</Subtitle>
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">VC Panel: Investing, Due Diligence, and Strategic Partnerships</Subtitle>
+                    <Text>Co-sponsored by <Link href="https://collegiate.vc/penn">Penn Undergraduate Capital Partners</Link></Text>
+                    <EventWithDateInfo date="APR 13 5 PM EST"
+                                        description={['Kimberly Zou @', 'Energy Impact Partners https://www.linkedin.com/in/kimberly-zou/',
+                                                        '\n Ry Storey-Fisher @', 'Powerhouse Fund https://www.linkedin.com/in/ry-storey-fisher/',
+                                                        '\n Franz Hochstrasser @', 'Raise Green https://www.raisegreen.com/']}/>
+                </EventWithDate>
+                <Subtitle fontSize="0.6rem">Pitch Office Hours</Subtitle>
+                <EventNoDate>
+                    <Text>Sign up for mentor office hours.</Text><br/>
+                    <Button>Sign Up Here</Button>
+                </EventNoDate>
+            </EventGroup>
+
+            <div css={`text-align: center;
+                        border-style: solid;
+                        border-radius: 40px;
+                        width: 70vw;
+                        padding: 3vw;
+                        display: block;
+                        margin: 4vw auto 0`}>
+                <Text fontSize="2rem" roboto bold>First Round Judging <br/> Due 4/16 at 11:59 PM EST</Text><br/>
+                <Button href="https://tinyurl.com/PCVPrizeSubmission">Submit Your Decks!</Button>
+            </div>
+        </div>
+    )
+}
+
+const Finals = () => {
+    return (
+        <div css={`text-align: center;
+                   border-style: solid;
+                   border-radius: 40px;
+                   width: 70vw;
+                   padding: 3vw;
+                   display: block;
+                   margin: 4vw auto`}>
```

Comment:
If these styles are repeated anywhere consider extracting out in a styled component - generally dislike when this many css styles are hard-coded to a specific elem